lib/instrumentation/superagent/nr-hooks.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+module.exports = [
+  {
+    type: 'generic',
+    moduleName: 'superagent',
```

Comment:
you don't need this. you can just have `lib/instrumentation/superagent.js`. the nr-hooks is designed when you have packages you're instrumenting that have more than once package, etc